src/test/java/org/fcrepo/client/HeadBuilderTest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.client;
+
+import static java.net.URI.create;
+import static org.fcrepo.client.TestUtils.baseUrl;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.net.URI;
+
+import org.apache.http.client.methods.HttpRequestBase;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+/**
+ * @author escowles
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class HeadBuilderTest {
+
+    @Mock
+    private FcrepoClient client;
+
+    @Mock
+    private FcrepoResponse fcrepoResponse;
+
+    private HeadBuilder testBuilder;
+
+    private URI uri;
+
+    @Before
+    public void setUp() throws Exception {
+        when(client.executeRequest(any(URI.class), any(HttpRequestBase.class)))
+                .thenReturn(fcrepoResponse);
+
+        uri = create(baseUrl);
+        testBuilder = new HeadBuilder(uri, client);
+    }
+
+    @Test
+    public void testHead() throws Exception {
+        testBuilder.perform();
+
+        final ArgumentCaptor<HttpRequestBase> requestCaptor = ArgumentCaptor.forClass(HttpRequestBase.class);
+        verify(client).executeRequest(eq(uri), requestCaptor.capture());
+
+        final HttpRequestBase request = getRequest();
+        assertEquals(0, request.getAllHeaders().length);
```

Comment:
This was copied from the Get test, where it was in contrast to later tests that checked for certain headers to be present: https://github.com/fcrepo4-exts/fcrepo-java-client/blob/master/src/test/java/org/fcrepo/client/GetBuilderTest.java#L69-L77

In this case, maybe I should just fold the getRequest() method into the testHead() to simply verify that the request was executed?